app/views/partials/timeline.mustache

```diff
@@ -0,0 +1,8 @@
+<ul class="timeline">
+  {{#timeline}}
+    <li data-test='interview-timeline' class="timeline__item {{#current}}timeline__item--current{{/current}} {{#first}}timeline__item--start{{/first}} {{#last}}timeline__item--finish{{/last}} {{priorityClass}}">
```

Comment:
Maybe instead of 3 different parameters 
```
{{#current}}timeline__item--current{{/current}} {{#first}}timeline__item--start{{/first}} {{#last}}timeline__item--finish{{/last}}
```
we could have only one i.e.
```
{{#position}}timeline__item-{{position}}{{/position}}
```
This will need to be reflected in `dashboard-view-model`